### PR TITLE
Added file upload component

### LINKED ISSUE #281: Add file upload component to form elements page
Add the file upload component (https://app.zeplin.io/project/5d95c0263689f402745a6e3a/styleguide/components?seid=5d95c714f425c1029f3e7055) to the design system's form components page.

image

An example of this has already been used, so base styline off of that. The code can be found here...
https://github.com/wmcadigital/wmn-covid19-refund/tree/master/src/components/Form/Step3/UploadTicket

To see it working go to: https://wmn-covid19-refund.netlify.app/
Click start > paper ticket > monthy direct debit > scroll down to bottom of page to see.

### PR DESCRIPTION
**Pull request by houbly**
Fixes #281 

Ignore the change to src/wmnds/components/content-tiles/normal/_normal.scss it should be fine but it was a commit from the previous branch, not sure why it committed again.

### CODE REVIEW

File: src/wmnds/components/form-elements/file-upload/_file-upload.njk [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <label for="fileUpload" class="wmnds-btn wmnds-btn--primary wmnds-fe-file-upload__label">
+          Choose file
+          {#- Get SVG icon -#}
+          {{
+            wmndsIcon({
+              icon: 'general-paperclip',
+              class: 'wmnds-btn__icon wmnds-btn__icon--right'
+            }) | indent(4) | trim
+          }}
+        </label>
```
Comment (gldgrnt) [inline src/wmnds/components/form-elements/file-upload/_file-upload.njk:55]: The indentation of the svg tag might be a bit confusing.

image

**Review by gldgrnt — CHANGES_REQUESTED** [review]
All good except for one bit of indentation.

**Review by gldgrnt — APPROVED** [review]
Yeah, looks great!

**Review by catiarodriguescosta — APPROVED** [review]
Looking good. 👍

**Pull request merged**